charts/ceph-csi-rbd/README.md

```diff
@@ -50,6 +50,32 @@ After installation succeeds, you can get a status of Chart
 helm status "ceph-csi-rbd"
 ```
 
+### Upgrade Chart
+
+If you want to upgrade your Chart, use the following commands.
+```bash
+helm repo update ceph-csi
+helm upgrade --namespace ceph-csi-rbd ceph-csi-rbd ceph-csi/ceph-csi-rbd
+```
+
+For upgrading to a specific version, provide the flag `--version` and the
+version.
+
+**Do not forget to include your values**, if they differ from the default values.
+We recommend not to use `--reuse-values` in case there are new defaults AND
+compare your currently used values with the new default values.
+
+#### Known Issues Upgrading
+- When upgrading to version >=3.7.0, you might encounter an error that the
+  CSIDriver cannot be updated (#3397). This is due to the CSIDriver resource not
```

Comment:
CSIDriver can be mentioned as `CSI Driver" or even menion `RBD CSI Driver"

Reply:
I went for `CSIDriver` because this is what the k8s resource is named – or `csidriver.storage.k8s.io` in its longer form. https://kubernetes-csi.github.io/docs/csi-driver-object.html

I'll change it in the first instance but in the next two instances I explicitly mean the k8s resource/object which uses the concatenated spelling. Is this acceptable?